lib/squadster/accounts.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Squadster.Accounts do
+  import Ecto.Changeset
+
+  alias Ueberauth.Auth
+  alias Squadster.Repo
+  alias Squadster.Accounts.User
+
+  def current_user(conn) do
+    conn.assigns[:current_user]
+  end
+
+  def signed_in?(conn) do
+    !!current_user(conn)
+  end
+
+  def logout(conn) do
+    conn
+    |> current_user
+    |> User.auth_changeset
+    |> put_change(:auth_token, nil)
+    |> Repo.update
+  end
+
+  def list_users do
+    Repo.all(User)
+  end
+
+  def find_user(id) do
+    Repo.get_by(User, id: id)
```

Comment:
maybe more clear
`User |> Repo.get_by(id: id)`
same in 2 methods below

Reply:
Fixed